src/evm_context.rs

```diff
@@ -406,13 +406,14 @@ fn declare_zero_or_one(var: &SMTVariable) -> Vec<SMTStatement> {
     ]
 }
 
-pub fn encode_revert_unreachable(ssa: &mut SSATracker) -> SMTStatement {
+pub fn encode_revert_reachable(ssa: &mut SSATracker) -> SMTStatement {
     smt::assert(smt::neq(context().revert_flag.smt_var(ssa), 0))
 }
 
-pub fn encode_solc_panic_unreachable(ssa: &mut SSATracker) -> SMTStatement {
+pub fn encode_solc_panic_reachable(ssa: &mut SSATracker) -> SMTStatement {
     smt::assert(smt::and_vec(vec![
         smt::neq(context().revert_flag.smt_var(ssa), 0),
+        smt::neq(context().revert_data_32.smt_var(ssa), 0),
```

Comment:
I don't think we should put this here. It's enough for the signature to match.

Reply:
that's true